ops.c

```diff
@@ -0,0 +1,285 @@
+#include "squashfuse.h"
+#include "fuseprivate.h"
+#include "stat.h"
+#include "nonstd.h"
+
+#include <errno.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "fs.h"
+#include "ops.h"
+
+void hello_world ( )
+{
+	printf("Hello World \n");
+
+}
+
+static sqfs_err sqfs_hl_lookup(sqfs **fs, sqfs_inode *inode,
+                const char *path) {
+        bool found;
+
+        sqfs_hl *hl = fuse_get_context()->private_data;
+        *fs = &hl->fs;
+        if (inode)
+                *inode = hl->root; /* copy */
+
+        if (path) {
+                sqfs_err err = sqfs_lookup_path(*fs, inode, path, &found);
+                if (err)
+                        return err;
+                if (!found)
+                        return SQFS_ERR;
+        }
+        return SQFS_OK;
+}
+
+
+
+void *sqfs_hl_op_init(struct fuse_conn_info *conn
+#if FUSE_USE_VERSION >= 30
+			     ,struct fuse_config *cfg
+#endif
+			     ) {
+	return fuse_get_context()->private_data;
+}
+
+void sqfs_hl_op_destroy(void *user_data) {
+	sqfs_hl *hl = (sqfs_hl*)user_data;
+	sqfs_destroy(&hl->fs);
+	free(hl);
+}
+
+int sqfs_hl_op_getattr(const char *path, struct stat *st
+#if FUSE_USE_VERSION >= 30
+                              , struct fuse_file_info *fi
+#endif
+                              ) {
+        sqfs *fs;
+        sqfs_inode inode;
+        if (sqfs_hl_lookup(&fs, &inode, path))
+                return -ENOENT;
+
+        if (sqfs_stat(fs, &inode, st))
+                return -ENOENT;
+
+        return 0;
+}
+ int sqfs_hl_op_opendir(const char *path, struct fuse_file_info *fi) {
```

Comment:
Add a space between functions.
```suggestion
}

 int sqfs_hl_op_opendir(const char *path, struct fuse_file_info *fi) {
```